fibonacci/fib.py

```diff
@@ -0,0 +1,8 @@
+import future
+
+def fib(n):
+    a, b = 0, 1
+    while a < n:
+        print(a)
+        a, b = b, a+b
+fib(1000)
```

Comment:
new line. and use https://github.com/hacktoberfest17/programming/blob/master/fibonacci/fibonacci.cpp for input/output reference.